neurots/generate/algorithms/tmdgrower.py

```diff
@@ -56,7 +56,9 @@ def __init__(
         barSZ = np.min(get_lengths(self.ph_angles))
         stepSZ = self.params["step_size"]["norm"]["mean"]
         if stepSZ >= barSZ:
-            L.warning("Selected step size %f is too big for bars of size %f", stepSZ, barSZ)
+            if barSZ > 0.1:
+                self.params["step_size"]["norm"]["mean"] = barSZ
```

Comment:
Since we check for `stepSZ >= barSZ`, we should set the value to `barSZ * factor` with `0 < factor < 1` to make the warning disappear (though you set it to `debug` so it will disappear anyway but the line 60 will be executed each time instead of just when it is needed).